CMakeLists.txt

```diff
@@ -60,23 +55,14 @@ configure_file(cmake/QtHardMonVersion.h.in
 configure_file(cmake/QtHardMon.qrc.in
   "${PROJECT_BINARY_DIR}/QtHardMon.qrc" @ONLY)
 
-#The use of the QWT plotting library is optional
-if(QWT_FOUND)
-  SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DUSE_QWT=true")
-  INCLUDE_DIRECTORIES( SYSTEM ${QWT_INCLUDE_DIR} )
-else(QWT_FOUND)
-  SET( QWT_LIBRARY "")
-  message("QWT not found, plotting is disabled in this build.")
-endif(QWT_FOUND)
-
 ADD_EXECUTABLE(${PROJECT_NAME} src/main.cpp ${PROJECT_BINARY_DIR}/QtHardMon.qrc)
+
 #The static library as well as its auto-generated headers are needed to compile main
-if(NOT ${CMAKE_VERSION} VERSION_LESS "3.7")
-  ADD_DEPENDENCIES(${PROJECT_NAME} ${PROJECT_NAME}_STATIC ${PROJECT_NAME}_STATIC_autogen)
-endif()
+ADD_DEPENDENCIES(${PROJECT_NAME} ${PROJECT_NAME}_STATIC ${PROJECT_NAME}_STATIC_autogen)
```

Comment:
I think this line is superfluous since dependency is implied by target_link_libraries

Reply:
Removed